doc/source/reference/random/index.rst

```diff
@@ -0,0 +1,237 @@
+.. _numpyrandom:
+
+.. currentmodule:: numpy.random
+
+numpy.random
+============
+
+Numpy's random number routines produce psuedo random numbers using
+combinations of generators and distributions:
+
+* Generators: Objects that generate random numbers. These are typically
+  unsigned integer words filled with sequences of either 32 or 64 random bits.
+* Distributions: Objects that transform sequences of random bits from a
+  generator into sequences of numbers that follow a specific probability
+  distribution (such as uniform, Normal or Binomial) within a specified
+  interval.
+
+Since Numpy version 1.17.0 the `RandomGenerator` can be initialized with a
+number of different Random Number Generators (RNG)s, and exposes many
+different probability distributions. See
+`NEP 19 <https://www.numpy.org/neps/nep-0019-rng-policy.html>`_ for context on
+the updated random Numpy number routines. The legacy `mtrand.RandomState`
+random number routines are still available, but limited to a single generator.
+
+For convenience and backward compatibility, a single `mtrand.RandomState`
+instance's methods are imported into the numpy.random namespace, see
+:ref:`legacy` for the complete list.
+
+Quick Start
+-----------
+
+By default, `RandomGenerator` uses normals provided by
+`xoroshiro128.Xoroshiro128` which will be faster than the legacy methods in
+`mtrand.RandomState`
+
+.. code-block:: python
+
+  # As replacement for numpy.random.RandomState
+  from numpy import random
+  random.standard_normal()
+
+`RandomGenerator` can be used as a direct replacement for
+`~RandomState`, although the random values are generated by
+`~xoroshiro128.Xoroshiro128`. The `RandomGenerator` holds an instance of a RNG.
+It is accessable as ``gen.brng``.
+
+.. code-block:: python
+
+  # As replacement for RandomState()
+  from numpy.random import RandomGenerator
+  rg = RandomGenerator()
+  rg.standard_normal()
+
```

Comment:
It could be useful to add here something like:

'The above code is equivalent to:
from numpy.random import Xoroshiro128
rg = Xoroshiro128().generator
rg.standard_normal()'

It creates an additional step to understanding the concept of a default brng and using another brng as shown next when passing a seed to the MT brng.

Reply:
[TODO]: For me. I agree that this would also be nneficial.